src/pages/VariantPage.js

```diff
@@ -96,6 +174,32 @@ const VariantPage = ({ match }) => (
         ) : null;
       }}
     </Query>
+    <hr />
+    <Heading>Associated index variants</Heading>
+    <SubHeading>
+      Which index variants and studies are linked to this tag variant?
+    </SubHeading>
+    <Query query={associatedIndexesQuery}>
+      {({ loading, error, data }) => {
+        console.log('data', data);
+        return hasAssociatedIndexVariants(data) ? (
+          <AssociatedIndexVariantsTable data={data} />
```

Comment:
Maybe you could just pass `data.indexVariantsAndStudiesForTagVariant.rows` since the check `hasAssociatedIndexVariants` has been made?

Reply:
:thumbsup: